documentation/assemblies/assembly-cruise-control-concepts.adoc

```diff
@@ -0,0 +1,10 @@
+// This assembly is included in the following assemblies:
+//
+// master.adoc
+
+[id='cruise-control-concepts-{context}']
+= Cruise Control
+
+This chapter provides an overview of the {ProductName} Cruise Control and helps you get started using Cruise Control with {ProductName}.
```

Comment:
"the Strimzi Cruise Control [implementation, deployment or similar]"? 
or maybe something like: 
"This chapter provides an overview of the Kafka cluster balancing features provided by [Cruise Control](https://github.com/linkedin/cruise-control) and helps you get started using Cruise Control with {ProductName}"